src/fides/api/service/connectors/sql_connector.py

```diff
@@ -824,7 +824,7 @@ def getconn() -> pg8000.dbapi.Connection:
                 config.instance_connection_name,
                 "pg8000",
                 user=config.db_iam_user,
-                db=config.dbname,
+                db=config.dbname or "postgres",
```

Comment:
maybe we can make this a class attribute rather than a raw string? like `self.default_dbname` or something?